scim2-sdk-common/src/main/java/com/unboundid/scim2/common/messages/PatchOperation.java

```diff
@@ -1371,4 +1349,35 @@ public static PatchOperation create(final PatchOpType opType,
         throw new IllegalArgumentException("Unknown patch op type " + opType);
     }
   }
+
+
+  /**
+   * Validates the {@code value} of a patch operation when the operation is
+   * constructed.
+   *
+   * @param path    The attribute path.
+   * @param value   The node containing the attribute value that will be
+   *                analyzed.
+   * @param type    The type of patch operation.
+   *
+   * @throws ScimException  If the provided value is null or invalid.
+   */
+  private static void validateOperationValue(final Path path,
+      final JsonNode value, final PatchOpType type)
+          throws ScimException
+  {
+    if (value == null || value.isNull() ||
+            (value.isObject() && value.size() == 0))
+    {
+      throw BadRequestException.invalidSyntax(
+          "value field must not be null or an empty container");
```

Comment:
Would you update the exception message here to be more specific?

Reply:
That makes sense. I've updated it to the following:

> The patch operation value must not be null or an empty object